closure/goog/graphics/canvasgraphics.js

```diff
@@ -668,3 +668,13 @@ goog.graphics.CanvasGraphics.prototype.resume = function() {
     this.needsRedraw_ = false;
   }
 };
+
+
+/**
+ * Removes an element from the Canvas.
+ * @param {goog.graphics.Element} elem the element to remove.
```

Comment:
Sorry, I just noticed `goog.graphics.Element` is nullable here. While it doesn't look like it would have any negative effect, there's no use wasting cycles if `elem` is null. Can you return immediately in that case? Also, you can tighten the `@param` of `CanvasGroupElement`'s `removeElement` to be non-nullable.